internal/corazawaf/transaction.go

```diff
@@ -900,14 +900,19 @@ func (tx *Transaction) ProcessRequestBody() (*types.Interruption, error) {
 		return nil, nil
 	}
 
+	if tx.interruption != nil {
+		tx.WAF.Logger.Error("Calling ProcessRequestBody but there is a preexisting interruption")
+		return tx.interruption, nil
+	}
+
 	if tx.LastPhase >= types.PhaseRequestBody {
 		// Phase already evaluated
 		tx.WAF.Logger.Warn("ProcessRequestBody has already been called")
```

Comment:
I feel like `ProcessRequestBody should have already been called` is better as it is not necessary that is has been called, maybe they jumped from phase 1 to 3 directly?